rabbitmq/datadog_checks/rabbitmq/check.py

```diff
@@ -0,0 +1,29 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from six import PY2
+
+from datadog_checks.base import AgentCheck, ConfigurationError, is_affirmative
+
+
+class RabbitMQ(AgentCheck):
+    __NAMESPACE__ = 'teamcity'
+
+    def __new__(cls, name, init_config, instances):
+        # TODO: can we use init config here?
+        instance = instances[0]
+
+        if is_affirmative(instance.get('use_openmetrics', False)):
```

Comment:
This for now is just a copy/paste from teamcity, I defer to a follow-up PR to decide if this way to manage configs fits this integration.

Reply:
This is coming in the next PR actually!